src/normalizer.ts

```diff
@@ -145,6 +145,30 @@ rules.set('Transform const to singleton enum', schema => {
   }
 })
 
+rules.set('Transform nullable -> anyOf', schema => {
+  if (schema.nullable === true) {
+    delete schema.nullable
+
+    const copiedSchema = {...schema}
+
+    // This stuff should not be in `anyOf`.
+    delete copiedSchema.id
+    delete copiedSchema.nullable
+
+    // This stuff will be in `anyOf` instead.
+    delete schema.format
+    delete schema.type
```

Comment:
Can you remove all of these `delete`s? Unclear why you need them (156 is already included in 150; 155+159+160 I'm not sure what they're here for).

Reply:
@bcherny 

I think we need all of these deletes. After normalizing, we no longer want `nullable`, `format`, or `type` to appear in the root of the schema. That's what the `anyOf` is replacing.

For example, the following schema:

```json
{
  "id": "foo",
  "type": "integer",
  "format": "int32",
  "nullable": true
}
```

Should be normalized to this:

```
 {
  "id": "foo",
  "anyOf": [
    {
      "type": "integer",
      "format": "int32"
    },
    {
      "type": "null"
    }
  ]
}
```

Notice how `nullable` is gone, and `type` and `format` have moved into the `anyOf`